diff -Naur expat-2.2.1.orig/lib/internal.h expat-2.2.1/lib/internal.h
--- expat-2.2.1.orig/lib/internal.h	2017-06-17 13:07:38.000000000 -0300
+++ expat-2.2.1/lib/internal.h	2017-07-03 09:08:43.410114076 -0300
@@ -80,13 +80,14 @@
 # endif
 #endif
 
+#include "expat_external.h"
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
 
-void
+XMLPARSEAPI(void)
 align_limit_to_full_utf8_characters(const char * from, const char ** fromLimRef);
 
 
diff -Naur expat-2.2.1.orig/lib/libexpat.def expat-2.2.1/lib/libexpat.def
--- expat-2.2.1.orig/lib/libexpat.def	2017-06-17 13:07:38.000000000 -0300
+++ expat-2.2.1/lib/libexpat.def	2017-07-03 09:09:26.886115425 -0300
@@ -74,3 +74,4 @@
 ; added with version 2.1.1
 ; XML_GetAttributeInfo @66
   XML_SetHashSalt @67@
+  align_limit_to_full_utf8_characters @90@
